### PR TITLE
Review request

### PR DESCRIPTION
**Pull request by ByPort**

### CODE REVIEW

File: src/app/layouts/blank-layout/blank-layout.component.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.mdl-layout .mdl-layout__content {
+  padding: 16px;
```
Comment (franckevva) [inline src/app/layouts/blank-layout/blank-layout.component.scss:4]: maybe `1rem` will be better

File: src/app/pages/charts/line-chart-1/line-chart-1.component.ts [hunk 2]
```diff
@@ -1,249 +1,57 @@
...
+    this.yAxis = 'Y';
+    this.maxX = 10;
+
+    this.afterConfigure();
+
+    this.animatedData = [
       {
         values: [],
         key: 'Sin(x)',
-        color: COLORS.lightBlue,
```
Comment (franckevva) [inline src/app/pages/charts/line-chart-1/line-chart-1.component.ts:50]: why do you decide to remove colors?

File: src/app/pages/dashboard/line-chart/line-chart.component.ts [hunk 3]
```diff
@@ -1,267 +1,56 @@
...
+
+import { LineChartComponent as BaseLineChartComponent } from 'theme/components/line-chart';
+
+import { LineChartService } from './line-chart.service';
 
 @Component({
   selector: 'app-line-chart',
-  styleUrls: ['./line-chart.component.scss'],
-  templateUrl: './line-chart.component.html',
+  styleUrls: ['../../../../theme/components/line-chart/line-chart.component.scss'],
```
Comment (franckevva) [inline src/app/pages/dashboard/line-chart/line-chart.component.ts:12]: Is it possible to use more pretty path?

Comment (ByPort) [reply src/app/pages/dashboard/line-chart/line-chart.component.ts:12]: No, https://github.com/angular/angular-cli/issues/5930#issuecomment-299428834

Only tricks with `require`/`import`

File: src/app/pages/charts/line-chart-1/line-chart-1.component.ts (hunk 2 above)
Comment (ByPort) [reply src/app/pages/charts/line-chart-1/line-chart-1.component.ts:50]: There were not many entries in the file. I left this constant in some files.

I can make `Injectable` from this constant for whole app but I don't see necessity for this.

**Pull request merged**